lib/index.js

```diff
@@ -114,7 +114,7 @@ function fileShouldBeDeleted(key, whitelist) {
 }
 
 function buildDeleteMultiple(Bucket, keys) {
-  if (!keys || !keys.length) return;
+  if (!keys || !keys.length) return [];
```

Comment:
I think we can safely remove completely the early return. `keys` is always an array, and all works fine even when it's empty.

Reply:
Makes sense, done.